### PR TITLE
[check-in] Allow the getSession call to be made in parallel to the feature_toggles call

### PR DESCRIPTION
**Pull request by brdunfield**
## Description
This PR removes the hard coded wait for the feature_toggles call to complete before allowing the getSession call to be made. In practice this allows our app to load and be interactable much sooner.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#45775

## Testing done
I manually checked whether feature toggles would be correctly loaded and they still are.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/check-in/day-of/pages/Landing.jsx [hunk 1]
```diff
@@ -76,7 +78,8 @@ const Landing = props => {
         goToErrorPage();
       }
 
-      if (token) {
+      if (token && !sessionCallMade) {
```
Comment (mdewey) [inline src/applications/check-in/day-of/pages/Landing.jsx:81]: Do you need add this to the `useEffect`  dependency array?

Comment (brdunfield) [reply src/applications/check-in/day-of/pages/Landing.jsx:81]: VSCode complained when I didn't.

Comment (brdunfield) [reply src/applications/check-in/day-of/pages/Landing.jsx:81]: Oh whoops yes I added it to one but not the other, thanks for the catch

**Review by acrollet — APPROVED** [review]
🚀

**Pull request merged**